gapic-generator-java/pom.xml

```diff
@@ -476,5 +476,17 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>5.10.2</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <version>5.10.2</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Pending the resolution of this discussion: https://github.com/googleapis/sdk-platform-java/pull/2754#discussion_r1599122878.

Version may be managed from pom-parent